x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message.tsx

```diff
@@ -97,7 +97,7 @@ export function WelcomeMessage({
 
           <EuiSpacer size="m" />
 
-          <EuiTitle size={breakpoint === 'xl' ? 'm' : 'l' ? 'm' : 's'}>
+          <EuiTitle size={breakpoint === 'xl' ? 'm' : breakpoint === 'l' ? 'm' : 's'}>
```

Comment:
Same here: I think it looks better (didn't test it though)
```suggestion
          <EuiTitle size={breakpoint === 'xl' || breakpoint === 'l' ? 'm' : 's'}>
```